.travis.yml

```diff
@@ -1,19 +1,35 @@
-language: ruby
-cache: bundler
+# Use Travis's cointainer based infrastructure
+sudo: false
+
 notifications:
   webhooks:
     urls:
       - https://webhooks.gitter.im/e/048b9bd9c30d5618d0a5
     on_success: change # options: [always|never|change] default: always
     on_failure: always # options: [always|never|change] default: always
     on_start: false # default: false
-rvm:
-  - 2.3.1
+addons:
+  apt:
+    sources:
+      - chef-current-precise
+    packages:
+      - chefdk
+
+# Don't `bundle install`
+install: echo "skip bundle install"
 
-bundler_args: --without integration development
+branches:
+  only:
+    - master
+
+# Ensure we make ChefDK's Ruby the default
 before_script:
-  - bundle exec berks
+  - eval "$(/opt/chefdk/bin/chef shell-init bash)"
+  # We have to install chef-sugar for ChefSpec
+  - /opt/chefdk/embedded/bin/chef gem install chef-sugar
```

Comment:
We can actually nuke this part. This came over from the cookbook I copied it from